### ISSUE TITLE
Getting indent error when running on fresh install.

### ISSUE THREAD

**Issue by namelessperson0** [rivermont/spidy#84]
Getting the below error
```
  File "crawler.py", line 217
    package='reppy')
    ^
IndentationError: unexpected indent
```
when running ` python crawler.py `

Incidentally, that line was changed recently in https://github.com/rivermont/spidy/commit/3a62f5edb8f03a87ef858e0cc27f8757097d1979

**Comment by paraita** [comment]
I confirm that fixed the issue for me 👍

**Comment by rivermont** [comment]
Thanks for the catch @namelessperson0. However we shouldn't remove the `package` arg as it's there for a reason. The problem was the extra parenthesis at the end of the line.

Fixed in 59e124d.
